style.css

```diff
@@ -66,6 +116,87 @@ h1 span:last-child {
     color: #36b37e;
 }
 
+#contact-box {
+    display: flex;
+    justify-content: center;
+}
+
+.card-container {
+    display: grid;
+    grid-template-columns: 100%;
+    grid-row-gap: 1rem;
+}
+
+.card {
+    border: 1px solid rgb(179, 177, 177);
+    border-radius: 8px;
+    padding-bottom: 1rem;
+}
+
+.card h1 {
+    font-size: 1.5rem;
+    margin: 1rem;
+}
+
+.card .tech {
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+    margin: 1rem;
+}
+
+.card .tech-tag {
+    background-color: #EBF0EE;
+    color: #3A4A42;
+    padding: 8px 12px;
+    margin: 0.3rem;
+    border-radius: 4px;
+    font-weight: 600;
+    font-size: 0.8rem;
+}
+
+div>p {
+    font-size: 1.2rem;
+    padding: 0.6rem 0;
+}
+
+.about-me-card {
+    background-color: #EBF0EE;
+}
+
+.card .about-me-tag {
+    background-color: white;
+    color: #36B37E;
+    padding: 8px 12px;
+    margin: 0.3rem;
+    border-radius: 8px;
+    font-weight: 600;
+    font-size: 0.8rem;
+}
+
+.card button, .btn-green {
+    font-size: 1.1rem;
+    background-color: #36B37F;
+    color: white;
+    border: 0;
+    padding: 0.8rem 1rem;
+    border-radius: 4px;
+}
```

Comment:
- [ ] Please consider adding all the button styling: Active, Hover and Pressed as seen in the design template you've chosen. This is to ensure that your button interactions are the same with your chosen template. See the button styling from your chosen template below:

<img width="217" alt="Loom_zktR6Ne5nZ" src="https://user-images.githubusercontent.com/81468675/148405053-f8631d78-d59e-4241-9195-64e0b54e4edf.png">